Statut.md

```diff
@@ -0,0 +1,169 @@
+
+§20 Członkowie zarządu #DoDyskusji
+1. Członkowie Zarządu Fundacji powoływani i odwoływani są przez Fundatorów w drodze zgodnego oświadczenia na czas wskazany w akcie powołania.
+2. Fundator może zostać członkiem Zarządu Fundacji.
+3. Zarząd Fundacji liczy od ... do ... (???) członków.
+4. Członkowie Zarządu Fundacji mogą otrzymywać wynagrodzenie z tytułu pełnienia swoich funkcji.
+5. Członkostwo w Zarządzie Fundacji ustaje w przypadku odwołania, pisemnej rezygnacji z członkostwa lub śmierci Członka.
+
+§21 Zadania zarządu
+1. Zarząd Fundacji kieruje działalnością Fundacji i reprezentuje ją na zewnątrz.
+2. Do zadań Zarządu należy w szczególności:
+	1) Kierowanie bieżącą działalnością Fundacji;
+	2) Podejmowanie decyzji w sprawie prowadzenia działalności gospodarczej;
+	3) realizacja celów statutowych Fundacji,
+	4) uchwalanie rocznych i wieloletnich planów działania Fundacji,
+	5) przygotowywanie i uchwalanie sprawozdań z działalności Fundacji,
+	6) zarządzanie majątkiem Fundacji.
+3. Zarząd Fundacji podejmuje decyzje na posiedzeniach w formie uchwał zwykłą większością głosów.
+4. O posiedzeniu muszą być powiadomieni wszyscy członkowie Zarządu z wyprzedzeniem co najmniej 7 dni kalendarzowych.
+5. Zarząd może powoływać pełnomocników do kierowania wyodrębnioną sferą spraw należących do zadań Zarządu Fundacji.  
+
+§22 Prezes i wiceprezes
+1. Zarządowi przewodniczy Prezes Zarządu Fundacji, wskazywany przez Fundatorów w akcie powołania Zarządu.
+2. Prezes Zarządu Fundacji zwołuje posiedzenia Zarządu Fundacji z własnej inicjatywy, bądź na wniosek każdego z pozostałych członków Zarządu Fundacji, nie rzadziej niż raz na pół roku.
+3. Prezes Zarządu Fundacji może powołać Wiceprezesa Zarządu Fundacji spośród pozostałych członków Zarządu, który wykonuje obowiązki Prezesa podczas jego niemożności działania lub w zakresie mu powierzonym.
+
+§23 Składanie oświadczeń woli i zwykły zarząd
+1. Do składania oświadczeń woli w imieniu Fundacji w sprawach majątkowych i niemajątkowych w zakresie zwykłego zarządu uprawniony jest Prezes Zarządu Fundacji samodzielnie.
+2. Do składania oświadczeń woli w imieniu Fundacji w sprawach majątkowych i niemajątkowych w zakresie przekraczającym zwykły zarząd uprawnieni są łącznie wszyscy Członkowie Zarządu Fundacji.
+3. Sprawy przekraczające zwykły zarząd:
+	a) dokonanie czynności prawnej nie wynikających z planów działania lub planów finansowych Fundacji;
+	b) dokonanie czynności prawnej o wartości przekraczającej 10 000 (słownie: dziesięć tysięcy PLN) zł, przy czym waluty obce należy przeliczać według kursu średniego NBP z dnia dokonania czynności prawnej;
```

Comment:
Czy możemy obniżyć wartość do 2 lub 3 tysięcy? Wtedy pewnie każdy członek zarządu będzie mógł podejmować takie czynności.

Reply:
1. To nie tak działa. Jak zrobisz wartość 2 tys. to każda decyzja o tej wartości będzie wymagał głosowania całego zarządu BARDZO OTWARTE